draft-pda-protocol.md

```diff
@@ -563,6 +569,27 @@ The leader responds to well-formed requests to `[leader]/upload` with status 200
 and an empty body. Malformed requests are handled as described in
 {{pa-error-common-aborts}}.
 
+### Upload Extensions {#upload-extensions}
+
+Each PAUploadReq carries a list of extensions that clients may use to convey
+additional, authenticated information in the report. Each extension is a tag-length
```

Comment:
```suggestion
additional, authenticated[TODO: Specify how] information in the report. Each extension is a tag-length
```

Reply:
The contents of the extensions are presumably fed into the AAD, so closing.